src/test/rmc/Pointers_OtherTypes/main.rs

```diff
@@ -21,7 +21,7 @@ fn main() {
     make_true(&mut z);
     assert!(z);
 
-    let mut a: f32 = rmc::nondet();
+    let mut a: f32 = unsafe { rmc::any_raw() };
```

Comment:
Should this be `rmc::any()`?

Reply:
Yes. Sorry, I fixed this before I implemented the Invariant for f32.